WordPress/src/main/java/org/wordpress/android/ui/reader/views/ReaderExpandableTagsView.kt

```diff
@@ -0,0 +1,154 @@
+package org.wordpress.android.ui.reader.views
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewTreeObserver.OnPreDrawListener
+import com.google.android.material.chip.Chip
+import com.google.android.material.chip.ChipGroup
+import org.wordpress.android.R
+import org.wordpress.android.WordPress
+import org.wordpress.android.models.ReaderTag
+import org.wordpress.android.models.ReaderTagList
+import org.wordpress.android.ui.utils.UiHelpers
+import org.wordpress.android.util.DisplayUtils
+import javax.inject.Inject
+
+class ReaderExpandableTagsView @JvmOverloads constructor(
```

Comment:
Great job on this class!!! I can't think of a better approach than using `addOnPreDrawListener`. So I'd go for it. 

The only thing I'd consider is to move the logic into a testable helper class - some things could be perhaps moved even into a uiState, but I'm not 100% it's necessary/desired.

Reply:
> move the logic into a testable helper class
Agree. I was thinking about how to make it testable. VM wasn't a right choice. I'll consider using a helper class.

Reply:
Adde todo note [here](https://github.com/orgs/wordpress-mobile/projects/63#card-43363060)